src/Service/Taf/NoaaService.php

```diff
@@ -0,0 +1,109 @@
+<?php
+declare(strict_types=1);
+
+namespace App\Service\Taf;
+
+use App\Traits\ZMQContextTrait;
+use Cake\Console\ConsoleIo;
+use Cake\Datasource\ModelAwareTrait;
+use Cake\Http\Client;
+
+/**
+ * @property \App\Model\Table\AirportsTable $Airports
+ * @property \App\Model\Table\TafTable $Taf
+ */
+class NoaaService
+{
+    use ModelAwareTrait;
+    use ZMQContextTrait;
+
+    /**
+     * @var string
+     */
+    protected const _NOAA_URL = 'https://aviationweather.gov/adds/dataserver_current/httpparam?dataSource=tafs&requestType=retrieve&format=xml&mostRecentForEachStation=true&hoursBeforeNow=0&stationString=';
```

Comment:
Set it to `https://aviationweather.gov/adds/dataserver_current/httpparam`.
See the comment on `NoaaService::_fetchTaf`